include/deal.II/numerics/fe_field_function.h

```diff
@@ -261,6 +262,14 @@ namespace Functions
     virtual void vector_value_list (const std::vector<Point< dim > >     &points,
                                     std::vector<Vector<typename VectorType::value_type> > &values) const;
 
+    /**
+     * Parallel version of vector_value_list
+     */
+    virtual void vector_value_list (const std::vector<Point< dim > >                      &points,
+                                    std::vector<Vector<typename VectorType::value_type> > &values,
+                                    const std::vector< BoundingBox<dim> >                 &local_bboxes) const;
```

Comment:
This implementation defeats the purpose of `FEFieldFunction`.

Notice that `FEFieldFunction` is derived from `Function`, and the method `vector_value_list` is overloaded from the base class. 

Therefore you have to find a different way to inform the class about the `local_bboxes` (for example at construction time, or attaching a signal to the triangulation).

Reply:
That is, if you want `FEFieldFunction` to be usable  where a `Function` object is required.

Reply:
Agree with Luca